### ISSUE TITLE
2.0.4.4

### ISSUE THREAD

**Issue by shawnp30** [MarlinFirmware/Marlin#16992]
using btt gtr v1.0 with 2.0.4.4 when enabling sdcard support i get errors with sd card string. when i disable eeprom and sd card supoort marlin compiles 
![image](https://user-images.githubusercontent.com/20215592/75390782-df305400-58b6-11ea-8fe0-897f99814745.png)

**Comment by szepnorbee** [comment]
+1 with BOARD_BTT_SKR_V1_3

**Comment by GMagician** [comment]
Try bugfix

**Comment by GMagician** [comment]
#16990 duplicate

**Comment by shawnp30** [comment]
tried bug fix still error

**Comment by freeasabeer** [comment]
In file queue.cpp line 530, replace MSG_SD_ERR_READ by STR_SD_ERR_READ

**Comment by lesstutrey** [comment]
freeasabeer that sort of works.. but just leads to new compile errors in planner.cpp so..

**Comment by freeasabeer** [comment]
Not on my side: melzi_optiboot with Creality Ender 3 configuration, the build ends successfully with the fix I proposed.

**Comment by oscar6502** [comment]
freeasabeer, works for me, thanks!

**Comment by wkelley70** [comment]
> In file queue.cpp line 530, replace MSG_SD_ERR_READ by STR_SD_ERR_READ

Worked for me with a BTT SKR 1.3.  Thanks.

**Comment by thisiskeithb** [comment]
This has been fixed in `bugfix-2.0.x`:
https://github.com/MarlinFirmware/Marlin/blob/bafe8d920a780bc5f5c0ad41437a09ffa823432d/Marlin/src/gcode/queue.cpp#L530

**Comment by Scope666** [comment]
> This has been fixed in `bugfix-2.0.x`:
> https://github.com/MarlinFirmware/Marlin/blob/bafe8d920a780bc5f5c0ad41437a09ffa823432d/Marlin/src/gcode/queue.cpp#L530

New error after applying this fix:

```
Marlin\src\module\planner.cpp: In static member function 'static bool Planner::_populate_block(block_t*, bool, const abce_long_t&, const xyze_pos_t&, feedRate_t, uint8_t, const float&)':
Marlin\src\module\planner.cpp:2407:19: error: 'maxj' was not declared in this scope
                   maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
                   ^~~~
Marlin\src\module\planner.cpp:2407:19: note: suggested alternative: 'fmaxl'
                   maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
                   ^~~~
                   fmaxl
Marlin\src\module\planner.cpp:2407:36: error: 'axis' was not declared in this scope
                   maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
                                    ^~~~
Marlin\src\module\planner.cpp:2407:36: note: suggested alternative: 'exit'
                   maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
                                    ^~~~
                                    exit
Marlin\src\module\planner.cpp:2396:23: warning: unused variable 'extra_xyjerk' [-Wunused-variable]
       constexpr float extra_xyjerk = 0;
                       ^~~~~~~~~~~~
Compiling .pio\build\LPC1768\src\src\module\servo.cpp.o
Compiling .pio\build\LPC1768\src\src\module\stepper.cpp.o
*** [.pio\build\LPC1768\src\src\module\planner.cpp.o] Error 1
```

**Comment by Kokorone** [comment]
> > This has been fixed in `bugfix-2.0.x`:
> > https://github.com/MarlinFirmware/Marlin/blob/bafe8d920a780bc5f5c0ad41437a09ffa823432d/Marlin/src/gcode/queue.cpp#L530
> 
> New error after applying this fix:
> 
> ```
> Marlin\src\module\planner.cpp: In static member function 'static bool Planner::_populate_block(block_t*, bool, const abce_long_t&, const xyze_pos_t&, feedRate_t, uint8_t, const float&)':
> Marlin\src\module\planner.cpp:2407:19: error: 'maxj' was not declared in this scope
>                    maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
>                    ^~~~
> Marlin\src\module\planner.cpp:2407:19: note: suggested alternative: 'fmaxl'
>                    maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
>                    ^~~~
>                    fmaxl
> Marlin\src\module\planner.cpp:2407:36: error: 'axis' was not declared in this scope
>                    maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
>                                     ^~~~
> Marlin\src\module\planner.cpp:2407:36: note: suggested alternative: 'exit'
>                    maxj = (max_jerk[axis]          // mj : The max jerk setting for this axis
>                                     ^~~~
>                                     exit
> Marlin\src\module\planner.cpp:2396:23: warning: unused variable 'extra_xyjerk' [-Wunused-variable]
>        constexpr float extra_xyjerk = 0;
>                        ^~~~~~~~~~~~
> Compiling .pio\build\LPC1768\src\src\module\servo.cpp.o
> Compiling .pio\build\LPC1768\src\src\module\stepper.cpp.o
> *** [.pio\build\LPC1768\src\src\module\planner.cpp.o] Error 1
> ```

I have this bug already submitted under this issue link. It only occurs when you have CLASSIC_JERK enabled

[https://github.com/MarlinFirmware/Marlin/issues/16991](url)

**Comment by zonalimitatore** [comment]
> 
> 
> In file queue.cpp line 530, replace MSG_SD_ERR_READ by STR_SD_ERR_READ

Thanks.Work on atmega gt2560

**Comment by Danielstc91** [comment]
> In file queue.cpp line 530, replace MSG_SD_ERR_READ by STR_SD_ERR_READ

Also worked for me with BTT SKR 1.3.

**Comment by thinkyhead** [comment]
Thanks for the report. Adding an automated test for EEPROM off with SD on.

**Comment by wmmc01** [comment]
This: In file queue.cpp line 530, replace MSG_SD_ERR_READ by STR_SD_ERR_READ, also fixed the error for the BTT SKR V1.4 Turbo!
